modules/collect/src/main/java/com/opengamma/strata/collect/io/CsvOutput.java

```diff
@@ -222,6 +222,22 @@ public void writeLines(Iterable<? extends List<String>> lines, boolean alwaysQuo
     }
   }
 
+  /**
```

Comment:
Move method below lines methods. ie. group all the lines methods, then all the rows methods, then all the cell methods, then the file/iterator ones

Reply:
Done.